doc/tutorials.rst

```diff
@@ -80,6 +80,16 @@ algorithms using PennyLane and near-term quantum hardware.
     :figure: ../examples/figures/quantum_natural_gradient/qng_optimization.png
     :description: :ref:`quantum_natural_gradient`
 
+.. customgalleryitem::
+    :tooltip: Perform QAOA for MaxCut
+    :figure: ../examples/figures/qaoa_maxcut_partition.png
+    :description: :ref:`qaoa_maxcut`
+    
+.. customgalleryitem::
+    :tooltip: Barren plateaus in quantum neural networks
```

Comment:
How come this is showing up here?

Reply:
I accidentally put some whitespace, it's gone now